export/views.py

```diff
@@ -5,13 +5,17 @@
     response,
     views,
     viewsets,
+    status,
 )
 
 from export.serializers import ExportSerializer
 from export.models import Export
 from project.models import Project
 
-from export.tasks_entries import export_entries
+from export.tasks import export_entries, export_assessment
+
+import logging
+logger = logging.getLogger('django')
```

Comment:
Is this used?

Reply:
It was used, later I removed logs and forget this one. My bad.